### PR TITLE
Allow to pass nextRun in update job

### PR DESCRIPTION
**Pull request by iwiznia**
Hold https://github.com/Expensify/Bedrock/pull/799

Needed for Expensify/Expensify#131401

Tests in the web PR

### CODE REVIEW

File: src/Jobs.php [hunk 1]
```diff
@@ -271,7 +271,7 @@ public function getJobs(string $name, int $numResults, array $params = []): arra
      *
      * @return array
      */
-    public function updateJob($jobID, $data, $repeat = null, ?int $priority = null)
+    public function updateJob($jobID, $data, $repeat = null, ?int $priority = null, ?string $nextRun = null)
```
Comment (alex-mechler) [inline src/Jobs.php:274]: Need to add the new param to the doc comment

Comment (iwiznia) [reply src/Jobs.php:274]: Why? It won't give much more information

Comment (alex-mechler) [reply src/Jobs.php:274]: Every other parameter for this already has them, regardless of if they add information, so it feels odd to not have it. Willing to have this a NAB though 🤷‍♂️

Comment (iwiznia) [reply src/Jobs.php:274]: Yeah, our stance is add it if it adds more information than the code... I could add a comment about it, but seems simple enough that it does not need one IMO. Happy to add it if you think it is.

**Comment by iwiznia** [comment]
Oh actually we don't need to hold... this will do nothing till bedrock is deployed

**Pull request merged**